pkg/shell/nav.tsx

```diff
@@ -195,7 +195,7 @@ export class CockpitNav extends React.Component {
         return (
             <>
                 <SearchInput placeholder={_("Search")} value={this.state.search} onChange={(_, search) => this.setState({ search })} onClear={() => this.setState({ search: "" })} className="search" />
-                <Nav onSelect={this.onSelect} theme="dark">
+                <Nav theme="dark">
```

Comment:
The "shell: Fix bugs found by TypeScript" commit still talks about Location.reload(), but now doesn't actually change that any more. (The other two issues are fine)